GooglePlayInstant/Editor/PlayInstantEditorMenu.cs

```diff
@@ -68,7 +68,37 @@ private static void SetUpPlayInstantSdk()
             PlayInstantSdkInstaller.SetUp();
         }
 
-        [MenuItem("PlayInstant/Build and Run #%r", false, 300)]
+        [MenuItem("PlayInstant/Quick Deploy/AssetBundle Creation...", false, 300)]
+        private static void AssetBundleCreationSettings()
+        {
+            PlayInstantQuickDeployWindow.ShowWindow(PlayInstantQuickDeployWindow.BundleCreateSelect);
+        }
+
+        [MenuItem("PlayInstant/Quick Deploy/AssetBundle Deployment...", false, 301)]
+        private static void AssetBundleDeploymentSettings()
+        {
+            PlayInstantQuickDeployWindow.ShowWindow(PlayInstantQuickDeployWindow.BundleDeploySelect);
+        }
+
+        [MenuItem("PlayInstant/Quick Deploy/AssetBundle Verification...", false, 302)]
+        private static void AssetBundleVerificationSettings()
+        {
+            PlayInstantQuickDeployWindow.ShowWindow(PlayInstantQuickDeployWindow.VerifyBundleSelect);
+        }
+
+        [MenuItem("PlayInstant/Quick Deploy/Loading Screen...", false, 303)]
+        private static void LoadingScreenSettings()
+        {
+            PlayInstantQuickDeployWindow.ShowWindow(PlayInstantQuickDeployWindow.LoadingScreenSelect);
+        }
+
+        [MenuItem("PlayInstant/Quick Deploy/Build APK...", false, 304)]
+        private static void BuildAPKSettings()
```

Comment:
Nit: "Apk" instead of "APK" for method names

https://msdn.microsoft.com/en-us/library/ms229043(v=vs.100).aspx